WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -137,21 +136,34 @@ private extension SupportTableViewController {
-        communityForumsSectionRows.append(SupportForumRow(title: LocalizedText.wpForumPrompt,
-                                                          action: nil,
-                                                          accessibilityIdentifier: "visit-wordpress-forums-prompt"))
-        communityForumsSectionRows.append(SupportForumButtonRow(title: LocalizedText.visitWpForumsButton,
-                                                                accessibilityHint: LocalizedText.visitWpForumsButtonAccessibilityHint,
-                                                                action: visitForumsSelected(),
-                                                                accessibilityIdentifier: "visit-wordpress-forums-button"))
-
-        let forumsSection = ImmuTableSection(headerText: LocalizedText.wpForumsSectionHeader,
-                                             rows: communityForumsSectionRows,
-                                             footerText: nil)
+    func tableViewModel() -> ImmuTable {
+
+        // Support section
+        let helpSection: ImmuTableSection?
+        switch SupportConfiguration.current() {
+        case .zendesk:
+            var rows = [ImmuTableRow]()
+            rows.append(HelpRow(title: LocalizedText.wpHelpCenter, action: helpCenterSelected(), accessibilityIdentifier: "help-center-link-button"))
+            rows.append(HelpRow(title: LocalizedText.contactUs, action: contactUsSelected(), accessibilityIdentifier: "contact-support-button"))
+            rows.append(HelpRow(title: LocalizedText.tickets, action: myTicketsSelected(), showIndicator: ZendeskUtils.showSupportNotificationIndicator, accessibilityIdentifier: "my-tickets-button"))
+            rows.append(SupportEmailRow(title: LocalizedText.contactEmail,
+                                        value: ZendeskUtils.userSupportEmail() ?? LocalizedText.emailNotSet,
+                                        accessibilityHint: LocalizedText.contactEmailAccessibilityHint,
+                                        action: supportEmailSelected(),
+                                        accessibilityIdentifier: "set-contact-email-button"))
+            helpSection = ImmuTableSection(headerText: LocalizedText.prioritySupportSectionHeader, rows: rows, footerText: nil)
+        case .forum:
+            var rows = [ImmuTableRow]()
+            rows.append(SupportForumRow(title: LocalizedText.wpForumPrompt,
+                                        action: nil,
+                                        accessibilityIdentifier: "visit-wordpress-forums-prompt"))
+            rows.append(SupportForumButtonRow(title: LocalizedText.visitWpForumsButton,
+                                              accessibilityHint: LocalizedText.visitWpForumsButtonAccessibilityHint,
+                                              action: visitForumsSelected(),
+                                              accessibilityIdentifier: "visit-wordpress-forums-button"))
+
```

Comment:
Nit: We could zap this extra space or add one above [line 154](https://github.com/wordpress-mobile/WordPress-iOS/pull/20163/files?diff=unified&w=0#diff-49dc6dec4a731852be5a9f38c831ff0ed74c648e5a412aafd19404fcbcb0bbcaR154) for consistency.

Reply:
I went with zapping this empty line 👍 Thanks for noticing!